# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/tests/mochitest.ini
# Commit: d554ca7850eb
# Full Hash: d554ca7850eb362a0482c9f4c19caf288e65efb1
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-08-04 04:34:13
# Regressor Bug: 1774704
# File Overlap Count: 1
# Description:
#   Bug 1774704 - part 0: Add automated tests of preserving selection while handling some commands r=m_kato
#   
#   The following patches touch the logic to restore selection after handling
#   commands.  However, it seems that they are not tested because I found some
#   regressions by manual testing, but didn't cause orange on tryserver.
# ==============================================================================

diff -r 86da8f2a28ea -r d554ca7850eb editor/libeditor/tests/mochitest.ini
--- a/editor/libeditor/tests/mochitest.ini	Wed Aug 03 23:58:51 2022 +0000
+++ b/editor/libeditor/tests/mochitest.ini	Thu Aug 04 00:21:53 2022 +0000
@@ -230,6 +230,7 @@
 [test_caret_move_in_vertical_content.html]
 [test_CF_HTML_clipboard.html]
 skip-if = os != 'mac' # bug 574005
+[test_cmd_absPos.html]
 [test_cmd_backgroundColor.html]
 [test_cmd_fontFace_with_tt.html]
 [test_cmd_paragraphState.html]